packages/lesswrong/lib/collections/tags/views.ts

```diff
@@ -98,3 +98,25 @@ Tags.addView('suggestedFilterTags', terms => {
 });
 
 ensureIndex(Tags, {deleted: 1, adminOnly: 1, suggestedAsFilter: 1, defaultOrder: 1, name: 1});
+
+Tags.addView('allLWWikiTags', terms => {
+  return {
+    selector: {
+      wikiOnly: viewFieldAllowAny,
```

Comment:
I'm not quite sure I understand what's going on here, but surprised that "lesswrongWikiImportSlug: {$exists:true}" works robustly here. I'm not too surprised it happens to work right now but don't we normally migrate things such that fields reliably exist, and wouldn't that then be all tags?

Reply:
For stringy fields, there isn't a way to "migrate things such that fields reliably exist". I wouldn't want to set this to an empty string, that would be worse.